.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea
```

Comment:
Can you please put this in your global ignore?  Not everybody uses intellij :-).